model/SimpleLicensing/Classes/LicenseExpression.md

```diff
@@ -39,3 +39,6 @@ license expression as a combination of license objects.
 - customIdToUri
   - type: /Core/DictionaryEntry
   - minCount: 0
+- customIdToLicense
+  - type: /Core/ElementMap
+  - minCount: 0
```

Comment:
In `LicenseExpression`'s description, we may like to add a paragraph telling the preference of using `customIdToLicense` over `customIdToUri`.

This new paragraph can be around Line 20, after the para starts with "The syntax for a licen..." and before the para starts with "The ExpandedLicensing profile can ...".